### DISCUSSION TITLE
Example 19, SimpleMintToExample fails

### DISCUSSION THREAD

**Discussion by pcarvalho75** [bmresearch/Solnet#224] [Q&A]
Im running the examples but I fail to properly mint tokens by running the example 19 as provided in the solnet.

Im getting "System.NullReferenceException: 'Object reference not set to an instance of an object.' Solnet.Rpc.Core.Http.RequestResult<T>.Result.get returned null." on line 159 of TransactionBuilderExample.cs

Could you please double-check on this?

**Comment by mariomatic** [comment]
Add `initalAccount `as a signer. The Memo Program requires the account to be a signer.

**Comment by pcarvalho75** [comment]
Dear Mario,

Thanks for your reply.  I have made some progress and managed to mint an
NFT using metaplex as per the examples provided in your project.

Here is a link to the mint.  Please note the json points to an image.
https://explorer.solana.com/address/2PkoVYtpnJFJe3uxEBCoB3de2K4BQjbzAyBsRA1JCY93/metadata?cluster=devnet

I can’t get the image to show up in the Solana explorer, as for example, in
this other NFT done by someone else:
https://explorer.solana.com/address/5AAEo1Z4rK4LkrKJXYFjxQ6hmUgvse6qdtMDr1ihbh2J

I don’t know where to start. would appreciate any help.  I also can't get
the master edition information to appear as per image.

Any suggestions?

[image: image.png]

Many thanks,
Paulo



On Thu, Oct 7, 2021 at 4:29 AM Mario Matic ***@***.***> wrote:

> Add initalAccount as a signer. The Memo Program requires the account to
> be a signer.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/bmresearch/Solnet/discussions/224#discussioncomment-1440798>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AB4SFLRYESXZ42NPH4JBDS3UFVD6BANCNFSM5FQGTL4Q>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
-----------------------------
Paulo Costa Carvalho, PhD
Laboratory for Structural and Computational Proteomics
Carlos Chagas Institute, Fiocruz - PR, Brazil
Curriculum Lattes: http://lattes.cnpq.br/9300734296482577

Journal of Proteomics - Executive Editor

**Comment by mariomatic** [comment]
everything's ok. Its the uri field. Its defined as http, so it gets blocked by the browser, because the explorer is https.

**Comment by pcarvalho75** [comment]
Thanks

**Comment by pcarvalho75** [comment]
I have minted a new token, now with https.  I have also installed https on
the website, but the Solana explorer still can't see my logo.

https://explorer.solana.com/address/2ztxGkMcRbH6jYWyfyTR1ZqSCDo5DFAaAXwnYX9qXu65/metadata?cluster=devnet

Any further suggestions?

Many thanks once again

On Tue, Oct 12, 2021 at 4:30 AM Mario Matic ***@***.***>
wrote:

> everything's ok. Its the uri field. Its defined as http, so it gets
> blocked by the browser, because the explorer is https.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/bmresearch/Solnet/discussions/224#discussioncomment-1462795>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AB4SFLUYTNXAFJGHTUZ2TD3UGPPXXANCNFSM5FQGTL4Q>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
-----------------------------
Paulo Costa Carvalho, PhD
Laboratory for Structural and Computational Proteomics
Carlos Chagas Institute, Fiocruz - PR, Brazil
Curriculum Lattes: http://lattes.cnpq.br/9300734296482577

Journal of Proteomics - Executive Editor

**Reply by hoakbuilds** [reply]
In order for the tokens to be displayed on the explorers they need to be added to the [Solana Token List](https://github.com/solana-labs/token-list/blob/main/src/tokens/solana.tokenlist.json)

Edit: disregard comment above, I didn't see it was an NFT. There might be something that's interfering with the way it fetches the image url from arweave, I haven't spent that much time looking into this so I can't say for sure

**Comment by pcarvalho75** [comment]
But as this is a metaplex NFT, from my understanding, the solana explorer
should be able to read the metadata and retrieve the logo; or am I wrong?



On Tue, Oct 12, 2021 at 7:14 PM hoak ***@***.***> wrote:

> In order for the tokens to be displayed on the explorers they need to be
> added to the Solana Token List
> <https://github.com/solana-labs/token-list/blob/main/src/tokens/solana.tokenlist.json>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/bmresearch/Solnet/discussions/224#discussioncomment-1467287>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AB4SFLR7IWZZG34KBZKOZELUGSXNJANCNFSM5FQGTL4Q>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
-----------------------------
Paulo Costa Carvalho, PhD
Laboratory for Structural and Computational Proteomics
Carlos Chagas Institute, Fiocruz - PR, Brazil
Curriculum Lattes: http://lattes.cnpq.br/9300734296482577

Journal of Proteomics - Executive Editor

**Comment by mariomatic** [comment]
Everything is ok on my side. Tried in Edge and Firefox. The logo ( ntf image ) is displayed, metadata is ok... Also checked it on SolScan,  everything is ok there too.

**Comment by pcarvalho75** [comment]
Hi there, thanks for the feedback. I managed to solve it after a couple of
hours. I had to include an.htacces to allow it.

Cheers
Paulo
